### PR TITLE
feat(Scripts): add images and svg loaders

### PR DESCRIPTION
**Pull request by jsomsanith**
**What is the problem this PR is trying to solve?**
No loaders for images and svg

**What is the chosen solution to this problem?**
Add the loaders

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/scripts/webapp/preset/config/webpack.config.js [hunk 1]
```diff
@@ -119,6 +119,20 @@ module.exports = ({ getUserConfig, mode }) => {
 						mimetype: 'application/font-woff',
 					},
 				},
+				{
+					test: /\.svg(\?v=\d+\.\d+\.\d+)?$/,
+					loader: 'url-loader',
+					options: {
+						name: 'assets/fonts/[name].[ext]',
```
Comment (jmfrancois) [inline packages/scripts/webapp/preset/config/webpack.config.js]: ?

File: packages/scripts/webapp/preset/config/webpack.config.js [hunk 2]
```diff
@@ -119,6 +119,24 @@ module.exports = ({ getUserConfig, mode }) => {
 						mimetype: 'application/font-woff',
 					},
 				},
+				{
+					test: /\.svg(\?v=\d+\.\d+\.\d+)?$/,
```
Comment (annilsson) [inline packages/scripts/webapp/preset/config/webpack.config.js]: Could you explain the query string here? Where does it come from?

Comment (jsomsanith) [reply packages/scripts/webapp/preset/config/webpack.config.js]: Oh in fact I copied/pasted that from TMC :D 
I'll fix it. Thanks

Comment (jsomsanith) [reply packages/scripts/webapp/preset/config/webpack.config.js]: Oh in fact I'm guilty of copy/pasting from TMC :D 
I'll fix it. Thanks

**Pull request merged**